tests/PySys/misc_features/custom_init_system/dummy_init.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# This script prints "command" and "service" to stdout.
+# USAGE
+# Input: $ ./dummy_init.sh restart mosquitto.service
+# Output: restart mosquitto.service
+
+COMMAND="$1"
+
+case "$COMMAND" in
+    is_available)
+        echo is_available | sudo tee -a /etc/tedge/dummy_init.out > /dev/null
```

Comment:
It's surprising to see `tee` combined with `> /dev/null`, the latter destroying the work of the former.

Why not:

```suggestion
        echo is_available >> /etc/tedge/dummy_init.out
```